### ISSUE TITLE
Collaboration Cycle for Debt Resolution, Streamlined Waiver 

### ISSUE THREAD

**Issue by HeatherRienks** [department-of-veterans-affairs/va.gov-team#57105]
### Verify information about your product or feature in the VFS Product Directory.

- [X] I have reviewed the [VFS Product Directory](https://depo-platform-documentation.scrollhelp.site/getting-started/vfs-product-directory) and confirm that my information is correct (or that I have taken appropriate actions to update or add my product information to the directory).

### VFS team name

Debt Resolution

### Product name

Financial Status Report (FSR)

### Feature name

Streamlined Waiver

### GitHub label for product

FSR Enhancements, FSR

### GitHub label for feature

StreamlinedWaiver

### When did/will you start working on this product/feature?

2022

### Will your work result in visible changes to the user experience?

Yes

### Are you doing research with VA.gov users?

Yes

### Will your work involve changes to...

Both

### Does your product/feature have Google Analytics tracking and a KPI dashboard in Domo?

Yes

### Do you need to capture any additional analytics or metrics?

Yes

### Product outline

https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/products/streamlined_waiver/product_outline.md

### Verify all relevant materials provided to Governance Team

- [X] I have provided all relevant and up-to-date links, screenshots, images, designs, etc. of the as-is version of the product

### Notify the Collaboration Cycle team of this ticket in the vfs-platform-support Slack channel.

- [X] I acknowledge that I must notify \#vfs-platform-support after submitting this issue.

**Comment by Tom-Davis** [comment]
@shiragoodman hello, wondering what touchpoints we need to address for the SW enhancement work. Thanks!

**Comment by shiragoodman** [comment]
Hey @Tom-Davis 

- Sitewide CAIA - I see a ticket was already submitted, so work with them as needed.

- Research Review - this was recommended bc a Yes response was given at kickoff when we asked, "Are you doing research with [VA.gov](http://va.gov/) users?" If you plan on working with Perigean, then you'll need to complete this.

- Midpoint Review - this is optional. Your choice.

- Analytics Request - this was recommended bc a Yes response was given at kickoff when we asked, "Do you need to capture any additional analytics or metrics?" If you plan on capturing additional metrics, then you'll need to complete this.

- Contact Center Review - I'd recommend checking with the Contact Center team via their Slack channel #vsp-contact-center-support to see if changes to your product guide/product video are needed.

- Staging Review - yes this is required

- Privacy Security Infra Readiness Review - yes this is required. I'd recommend synching with the Platform Security team ASAP. They have always recommended that every team/product have the review, but are starting to sway from that model. The sooner you engage, the sooner they might be able to tell you that it's not needed (or if it's needed, they can provide support earlier). I'd recommend reaching out to them via their Slack channel #platform-security


In short, if it's listed in your Collaboration Cycle Request ticket, then it's recommended for you to complete. All touchpoints, excluding Staging Review and Privacy, Security, Infrastructure Readiness Review, are optional.

**Comment by HeatherRienks** [comment]
@shiragoodman - thank you!
